t/plugin/traffic-split2.t

```diff
@@ -732,3 +732,80 @@ location /t {
 qr/1980, 1981, 1982, 1980, 1981, 1982, 1980, 1981, 1982/
 --- no_error_log
 [error]
+
+
+
+=== TEST 18: multi nodes with `node` mode to pass host
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/upstreams/1',
+                 ngx.HTTP_PUT,
+                 [[{
+                    "nodes": {
+                        "localhost:1979": 1000,
+                        "127.0.0.1:1980": 1
```

Comment:
I think it would be better if this port is different from the default port on the route, for example: `127.0.0.1:1981`. This is more obvious in the test results. Otherwise, it is easy for people to misunderstand that the `proxy request to 127.0.0.1:1980` in the test result hits the default route, but it is not.